acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/CmdLineParamsConfigFileImpl.java

```diff
@@ -295,7 +308,70 @@ private String createEth2SlashingProtectionArgs() {
     return yamlConfig.toString();
   }
 
-  private String createCommaSeparatedList(final List<String> values) {
+  private String awsSecretsManagerOptions() {
+    final StringBuilder yamlConfig = new StringBuilder();
+
+    if (signerConfig.getAwsSecretsManagerParameters().isPresent()) {
```

Comment:
Think this would be clearer if the if was outside in the createCmdLineParams method

Reply:
👍🏼